share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -136,7 +136,7 @@ default:
       --config-scope "${SPACK_CI_CONFIG_ROOT}/${SPACK_TARGET_PLATFORM}"
       --config-scope "${SPACK_CI_CONFIG_ROOT}/${SPACK_TARGET_PLATFORM}/${SPACK_TARGET_ARCH}"
       config blame > "${CI_PROJECT_DIR}/jobs_scratch_dir/spack.yaml.blame"
-    - spack -v  --color=always
+    - spack -d -v --color=always
```

Comment:
IMO it's worth running all spack commands in CI with `-d`, so I left this here.

Reply:
I thought this was something that slipped through, but I see it is not. I'm against this change, for this reason:
- https://gitlab.spack.io/spack/spack/-/jobs/16901136

The output is not readable, and is truncated because it exceeds limits. I'll submit a PR to revert that, and point to these comments for reference.